parsers/BR.py

```diff
@@ -116,8 +117,8 @@ def fetch_exchange(country_code1='BR', country_code2='UY', session=None):
     gd = get_data()
 
     data = {
-        'datetime': gd['Data'],
-        'sortedCountryCodes': 'BR->UY',
+        'datetime': parser.parse(gd['Data']),
```

Comment:
You don't need the dateutil dependency.
Simply call `.datetime` on the arrow object returned by `gd['Data']`

Reply:
I got it!